velox/expression/tests/ExprTest.cpp

```diff
@@ -2924,3 +2926,9 @@ TEST_F(ExprTest, flatNoNullsFastPath) {
   ASSERT_FALSE(exprSet->exprs()[0]->supportsFlatNoNullsFastPath())
       << exprSet->toString();
 }
+
+TEST_F(ExprTest, parseDecimalConstant) {
```

Comment:
nit: Does this test belong to DuckParserTest?

Reply:
`parseExpression` used below is implemented in this test and invokes `parse::parseExpr(text, options);` as well as `core::Expressions::inferTypes`.
So it fits better here as it tests the TypedExpr path as well.

Reply:
In hindsight, I will move this to DuckParserTest. There are no changes in `core::Expressions::inferTypes` to test.